### PR TITLE
Fix Dashboard URI returned by cluster.status()

### LINKED ISSUE #316: Incorrect Dashboard URI returned by cluster.status() method
### Describe the Bug
Ray Dashboard URI from hyperlink is incorrect.

image

### Steps to Reproduce the Bug

1. In a cluster go to the Open Datahub dashboard and create a Jupyter Notebook and run through one of the guided-demos i.e.: `0_basic_ray.ipynb`
2. When running through the demo, after running `cluster.status()` you can attempt to open the Ray Dashboard through the provided hyperlink as shown in the screenshot above. 

### What Have You Already Tried to Debug the Issue?

It seems the method is returning something along these lines:
```
https://jupyter-nb-kubeadmin-opendatahub.apps.cluster-name.zcanzs.s1.devshift.org/notebook/opendatahub/jupyter-nb-kubeadmin/files/codeflare-sdk/demo-notebooks/guided-demos/ray-dashboard-raytest-default.apps.cluster-name.zcanzs.s1.devshift.org?_xsrf=2%7C217c190f%7Cdb6c4b08e55569b200e2302511a46155%7C1
```

### Expected Behavior

We are expecting a link similar to this:
`http://ray-dashboard-raytest-default.apps.cluster-name.zcanzs.s1.devshift.org/`

### PR DESCRIPTION
**Pull request by ChristianZaccaria**
# Issue link
Closes #316 

# What changes have been made

This change fixes the issue where the hyperlink provided by the cluster.status() method is incorrect.

Previously the hyperlink provided resolved to a relative path as opposed to the absolute Ray Dashboard URI. For this reason, I prefixed `http://` to ensure the hyperlink points to the correct URI.

# Verification steps

1. In a cluster go to the Open Datahub dashboard and create a Jupyter Notebook
2. Uninstall the current codeflare-sdk with pip uninstall codeflare-sdk -y
3. Install the codeflare-sdk from this branch: You will need to run `poetry build` to generate a wheel of the SDK.
4. Upload the wheel to the Jupyter Notebook, and install the build with `pip install codeflare_sdk-0.0.0.dev0-py3-none-any.whl`.
5. Restart the Notebook Kernel
6. Run through one of the guided-demos i.e.: 0_basic_ray.ipynb
7. When running through the demo, after running cluster.status() you can attempt to open the Ray Dashboard through the provided hyperlink.

## Checks

- Testing Strategy
   - [x] Manual tests

### CODE REVIEW

**Review by KPostOffice — COMMENTED** [review]
One comment

File: src/codeflare_sdk/cluster/cluster.py [hunk 1]
```diff
@@ -544,7 +544,7 @@ def _map_to_ray_cluster(rc) -> Optional[RayCluster]:
     ray_route = None
     for route in routes["items"]:
         if route["metadata"]["name"] == f"ray-dashboard-{rc['metadata']['name']}":
-            ray_route = route["spec"]["host"]
+            ray_route = f"http://{route['spec']['host']}"
```
[the remark this answers is not in the document]
Comment (ChristianZaccaria) [reply src/codeflare_sdk/cluster/cluster.py]: Excellent one @KPostOffice , thanks! I made the changes now, and applied them to another function that was only taking `http` into account as well.

**Review by KPostOffice — APPROVED** [review]
/lgtm

**Review by MichaelClifford — APPROVED** [review]
LGTM

**Pull request merged**